Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -405,6 +405,20 @@ internal static void Update()
             if (!active)
                 return;
 
+            // Check for dead clients but exclude the host client because it
+            // doesn't ping itself and therefore may appear inactive.
+            if (NetworkManager.singleton.disconnectInactiveConnections)
```

Comment:
almost :)
this creates a circular reference where networkmanager uses networkserver, and networkserver uses networkmanager. that's a problem because it increases complexity and dependencies.

instead add a disconnectInactiveConnections flag to NetworkServer too.
and in NetworkManager.StartServer set networkserver.disconnectInactiveConnections to networkmanager.disconnectInactiveConnections :)

Reply:
Done